packages/@o3r/configuration/schematics/configuration-to-component/index.ts

```diff
@@ -36,6 +36,13 @@ const configProperties = [
 ];
 
 const checkConfiguration = (componentPath: string, tree: Tree) => {
+  const files = [
+    resolve(dirname(componentPath), `./${basename(componentPath, '.component.ts')}.fixture.ts`)
+  ];
+  if (files.some((file) => tree.exists(file))) {
+    throw new Error(`Unable to add fixture to this component because it already has at least one of these files: ${files.join(', ')}.`);
```

Comment:
I'm not sure to understand this part, so basically you cannot add the configuration if there already is a fixture ?

```suggestion
    throw new Error(`Unable to add configuration to this component because it already has at least one of these files: ${files.join(', ')}.`);
```